.github/scripts/createCommit.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Is this script under `.github` directory because that's what's required for GitHub Actions?

Reply:
not really...I'm just following examples on docs...don't worry about changes in `.github` yet...Things don't work properly just yet. I have a separate ticket working on this. I jsut figured I'll use this as a test. But I'm going to disable it before I merge this PR

Reply:
I see, okay. Might be good to do it as a separate PR next time 😊

Reply:
yea..sorry about this...oh actually, what I was thinking is if I fix the CI, then I don't have to manually fix the PR to `terra-helm`...that'll also prove CI works properly..so think I'm gonna try to get CI working properly before I merge this one (unless it takes too long to make CI work)